discovery/pkg/cmd/root.go

```diff
@@ -28,6 +29,8 @@ func init() {
 	// Get the root command properties and bind the config property in YAML definition
 	rootProps := RootCmd.GetProperties()
 	config.AddProperties(rootProps)
+
+	migrate.MatchAttrPattern("-hash")
```

Comment:
Does this need to be a regex? .*-hash? @tsjohns9

Reply:
i think the pattern matches it, maybe, I took a leap. I'll test it.  I saw all my '-hash' values come thru. But i'll verify

Reply:
-hash will work

Reply:
You will need to call `migrate.MatchAttr` and pass in the unique Apigee attributes so they can be moved over